api/Notification.json

```
@@ -933,11 +954,13 @@
             ],
             "safari_ios": {
               "version_added": "16.4",
-              "notes": "Notifications are supported in web apps saved to the home screen."
+              "partial_implementation": true,
+              "notes": "The <code>requestPermission()</code> is only supported in web apps saved to the home screen, and using a non-default display mode."
```

This is incomplete. I'm guessing it's either something like this (if it's relatively normal):

```suggestion
              "notes": "The parent <code>Notification</code> interface undefined unless the page is a web app saved to the home screen. The app's manifest must have a non-default <code>display</code> value."
```

or something like this, if they intercept this call specifically:

```suggestion
              "partial_implementation": true,
              "notes": "The returned promise rejects unless the page is a web app saved to the home screen. The app's manifest must have a non-default <code>display</code> value."
```

---

The `Notification` interface is undefined unless the app is saved to the home screen. So the first suggest applies here.